src/pygeoprocessing/routing/routing.pyx

```diff
@@ -934,8 +914,7 @@ def fill_pits(
                     # search to see if the fill has gone on too long
                     if search_steps > max_pixel_fill_count:
                         # clear the search queue and quit

```

Comment:
Remove comment about clearing search queue?

Reply:
That section does clear the search queue though... Can I leave it in? I'll change it to clear `pit_queue`?

Reply:
Changed to say `pit_queue` instead of search queue in 3979892